File: Python/Disk.py
class Disk:
    #atribut
    __type=""
    __capacity=0
    __price=0

    #constructor
    def __init__(self, type="", capacity=0, price=0):
        self.__type=type
        self.__capacity=capacity
        self.__price=price

    #Setter Getter
    def setType(self, type):
        self.__type=type
    
    def setCapacity(self, capacity):
        self.capacity=capacity

    def setPrice(self, price):
        self.__price=price

    def getType(self):
        return self.__type
    
    def getCapacity(self):
        return self.__capacity

    def getPrice(self):
        return self.__price

    #Method
    #Menampilkan atribut Disk
    def printDisk(self):
        print("Disk Type: " + self.getType())
        print("Disk Capacity: " + str(self.getCapacity()))
        print("Disk Price: " + str(self.getPrice()))

File: PHP/Disk.php
<?php
class Disk{
    //atribut
    private $type;
    private $capacity;
    private $price;

    //constructor
    function __construct($type, $capacity, $price) {
        $this->type=$type;
        $this->capacity=$capacity;
        $this->price=$price;
    }

    //Setter Getter
    function setType($type) { $this->type=$type; }
    function setCapacity($capacity) { $this->capacity=$capacity; }
    function setPrice($price) { $this->price=$price; }

    function getType() { return $this->type; }
    function getCapacity() { return $this->capacity; }
    function getPrice() { return $this->price; }

    //Method
    //Menampilkan atribut Disk
    function printDisk() {
        echo "Disk Type: " . $this->getType() . "<br>";
        echo "Disk Capacity: " . $this->getCapacity() . "<br>";
        echo "Disk Price: " . $this->getPrice() . "<br>";
    }

    function __destruct() {}
}
